event/src/args.rs

```diff
@@ -40,40 +45,58 @@ impl<'a, S: Signer> EventArgs<'a, S> {
         Self {
             signer,
             controllers: vec![signer.id()],
-            parent,
+            model: parent,
             sep: SEP,
         }
     }
 
     /// Parent stream id
     pub fn parent(&self) -> &StreamId {
```

Comment:
thanks for updating parent->model here!  Should we also update the name of this accessor method while we're at it?